arangod/Statistics/StatisticsFeature.cpp

```diff
@@ -609,25 +601,19 @@ StatisticsFeature::StatisticsFeature(Server& server)
       if (it.second != nullptr) {
         auto const& builder = *it.second;
         auto const& stat = statIt->second;
-        std::string statName(stat[0]);
-        auto const& statType = metricType(stat[1], true);
-        if (statType == "counter") {
-          statName += "_total";
-        }
-        [[maybe_unused]] auto const& statHelp = stat[2];
-        if (builder.name() != statName) {
+        if (builder.name() != stat[0]) {
           foundError = true;
           LOG_TOPIC("f66dd", ERR, Logger::STATISTICS)
               << "Statistic '" << it.first << "' has mismatching names: '"
-              << builder.name() << "' in statBuilder but '" << statName
```

Comment:
But in other place this code used stat[2] and stat[1].
And statName used here not because of naming. But because of v1/v2 metrics have different names.

And also I think we don't want write name, help, etc twice or more (stats map, declare macros, yaml docs, some others). We propose implementation idea: all metrics name, types and helps string generated from single source of true (yaml docs).
So in future we don't have stats (btw why its runtime map?)